packages/create-nx-workspace/src/utils/ci/ci-list.ts

```diff
@@ -5,3 +5,5 @@ export const ciList = [
   'bitbucket-pipelines',
   'gitlab',
 ] as const;
+
+export type CI = typeof ciList[number];
```

Comment:
Can we do the same thing for framework and the other enum options we provide in yargs?

Reply:
done.